src/state/useItineraries.ts

```diff
@@ -186,10 +186,13 @@ const useItineraries = () : [ItinerariesState, ItinerariesActions, ItinerariesSe
 
   const clear = () => dispatch(createClear())
 
-  // @TODO actually call the API instead of mocking it using a setTimeout.
-  const setChecked = (itineraryId: Id, checked:boolean) => setTimeout(() => {
-      dispatch(createSetChecked(itineraryId, checked))
-    }, 1000)
+  const setChecked = async (itemId: Id, checked:boolean) => {
+    const url = getUrl(`itinerary-items/${ itemId }`)
+    const [response, ] = await patch(url, { checked })
```

Comment:
Could be `const [response] = await patch(url, { checked })`. Is there a lint rule for this?

Reply:
I think there is:
https://eslint.org/docs/rules/comma-dangle

Will make a different PR to implement this rule.